lib/gooddata/bricks/pipeline.rb

```diff
@@ -87,6 +88,15 @@ def self.rollout_brick_pipeline
         RolloutBrick
         ])
       end
+
+      def self.hello_world_brick_pipeline
+        prepare([
+          DecodeParamsMiddleware,
+          LoggerMiddleware,
+          BenchMiddleware,
+          HelloWorldBrick
+        ])
```

Comment:
warning: Style/IndentArray: Indent the right bracket the same as the first position after the preceding left parenthesis. (rubocop)